README.md

```diff
@@ -80,6 +80,8 @@ function App() {
 render(<App />, document.getElementById('root'));
 ```
 
+> Replace `<APP_ID>` with your public app ID that you can find at Workspaces > Applications > Setup.
```

Comment:
It's missing the second part I've suggested:

```suggestion
> Replace `<APP_ID>` with your public app ID that you can find at Workspaces > Applications > Setup.
> In case you don't have an account yet, you can use the sandbox application ID `00000000-0000-0000-0000-000000000000`
```